callstats-java-sdk/src/main/java/io/callstats/sdk/messages/AuthorizeRequest.java

```diff
@@ -0,0 +1,46 @@
+package io.callstats.sdk.messages;
+
+/**
+*
+* @author Karthik Budigere
+*/
+public class AuthorizeRequest {
```

Comment:
I am accessing those classes from the main package, so i cannot remove the "public" access modifier.